### PR TITLE
ShopSearch Page

### PR DESCRIPTION
**Pull request by butalas**
### Description
Updates the ShopSearch Page

### Context
I fixed the absolute styling so everything is relative with Dimensions. A user can type into the search bar and press the search icon to go to the next page (it didn't work with the PrimaryButton so I used a TouchableOpacity). 

Since it is the very first page that shows up, the Montserrat fonts weren't loading for the page and it was giving me an error that I had used the fonts before loading them. To fix this, I put the FontLoad stuff into this file. I wasn't sure how else to resolve the error. 

### Issue
#91 

### Screenshot
simulator screen shot - iphone x - 2019-02-27 at 17 12 49

### CODE REVIEW

File: screens/ShopSearch.js [hunk 1]
```diff
@@ -1,110 +1,143 @@
...
+        <View style={styles.container}>
+          <OrangeBackground/>
+          <View style={styles.toolbar}>
+              <Image style={styles.cartIcon}
+                  source={require('./../assets/images/top_shopping_cart.png')} />
+              <Image style={styles.personIcon}
+                  source={require('./../assets/images/person.png')} />
+          </View>
+          <View style={styles.mainView}>
+            <Text style={styles.header}>Let's shopU!</Text>
```
Comment (ijemmao) [inline screens/ShopSearch.js]: Make sure to escape the apostrophe. GitHub is parsing it terribly.

File: screens/ShopSearch.js [hunk 2]
```diff
@@ -1,110 +1,143 @@
...
-        <View style={styles.searchLine}></View>
-        <Text style={styles.searchText}>Search for a Product</Text>
-        <Image source={require('./../assets/images/search_icon.png')}style={styles.searchIcon} />
-        <Image source={require('./../assets/images/top_shopping_cart.png')}style={styles.topShopCart} />
-        <Image source={require('./../assets/images/profile_head.png')}style={styles.profileHead} />
-      </View>
+      this.state.fontLoaded ? (
+        <View style={styles.container}>
+          <OrangeBackground/>
+          <View style={styles.toolbar}>
```
Comment (ijemmao) [inline screens/ShopSearch.js]: Use the `Toolbar` component

**Comment by butalas** [comment]
Hey! I just fixed the thing with the apostrophe and replaced my toolbar stuff with the `Toolbar` component.

File: screens/ShopSearch.js [hunk 3]
```diff
@@ -1,110 +1,127 @@
...
-        <View style={styles.searchLine}></View>
-        <Text style={styles.searchText}>Search for a Product</Text>
-        <Image source={require('./../assets/images/search_icon.png')}style={styles.searchIcon} />
-        <Image source={require('./../assets/images/top_shopping_cart.png')}style={styles.topShopCart} />
-        <Image source={require('./../assets/images/profile_head.png')}style={styles.profileHead} />
-      </View>
+      this.state.fontLoaded ? (
+        <View style={styles.container}>
+          <OrangeBackground/>
+          <View style={{marginTop: window.height*.015}}>
```
Comment (ijemmao) [inline screens/ShopSearch.js]: Why did you add this styling? You shouldn't have to add extra styling around the `Toolbar` component

**Comment by butalas** [comment]
Page should be done!
